src/handlers/major_change.rs

```diff
@@ -362,3 +417,183 @@ fn zulip_topic_from_issue(issue: &ZulipGitHubReference) -> String {
+            .context("unable to deserialize the metadata in major change acceptence job")?;
+
+        let now = Utc::now();
+
+        match process_seconded(&ctx, &major_change, now).await {
+            Ok(()) => {
+                tracing::info!(
+                    "{}: major change ({:?}) as been accepted, remove from the queue",
+                    self.name(),
+                    &major_change,
+                );
+            }
+            Err(err) if err.downcast_ref::<SecondedLogicError>().is_some() => {
+                tracing::error!(
+                    "{}: major change ({:?}) has a logical error (no retry): {err}",
+                    self.name(),
+                    &major_change,
+                );
+                // exit job succesfully, so it's not retried
+            }
+            Err(err) => {
+                tracing::error!(
+                    "{}: major change ({:?}) is in error: {err}",
+                    self.name(),
+                    &major_change,
+                );
+                return Err(err); // so it is retried
+            }
+        }
+
+        Ok(())
+    }
+}
+
+async fn process_seconded(
+    ctx: &super::Context,
+    major_change: &MajorChangeSeconded,
+    now: DateTime<Utc>,
+) -> anyhow::Result<()> {
+    if major_change.accept_at < now {
+        anyhow::bail!(SecondedLogicError::NotYetAcceptenceTime {
+            accept_at: major_change.accept_at,
+            now
+        });
+    }
+
+    let repo = ctx
+        .github
+        .repository(&major_change.repo)
+        .await
+        .context("failed retrieving the repository informations")?;
+
+    let config = crate::config::get(&ctx.github, &repo)
+        .await
+        .context("failed to get triagebot configuration")?;
+
+    let config = config
+        .major_change
+        .as_ref()
+        .ok_or(SecondedLogicError::NoMajorChangeConfig)?;
+
+    let issue = repo
+        .get_issue(&ctx.github, major_change.issue)
+        .await
+        .context("unable to get the associated issue")?;
+
+    if issue
+        .labels
+        .iter()
+        .any(|l| Some(&l.name) == config.concerns_label.as_ref())
+    {
+        anyhow::bail!(SecondedLogicError::ConcernsLabelSet);
+    }
+
+    if !issue.is_open() || issue.draft {
+        anyhow::bail!(SecondedLogicError::IssueNotReady {
+            draft: issue.draft,
+            open: issue.is_open()
+        });
+    }
+
+    if !issue.labels.iter().any(|l| l.name == config.accept_label) {
+        // Only post the comment if the accept_label isn't set yet, we may be in a retry
+        issue
+            .post_comment(
+                &ctx.github,
+                "The final comment period is now complete, this major change is now accepted.\n\nAs the automated representative, I would like to thank the author for their work and everyone else who contributed to this major change proposal."
```

Comment:
Since this is a bot closing a human-driven process, I would add a bit or wording along the line of "feel free to reopen if there open concerns"

Reply:
Sure, will add that.

Reply:
Opened https://github.com/rust-lang/triagebot/pull/2072